### ISSUE TITLE
Headless/Desktop does not deploy ovf on vSphere 6.5 7801515

### ISSUE THREAD

**Issue by FlosCarmeli** [orlikoski/Skadi#81]
I did not have an issue deploying the previous release of Skadi, but this version will not deploy with OVF. Also, I tried creating a VM and using the portable version to install as an ISO and it runs unbearably slow even with flash storage, 16CPU and 64GB of RAM. 

I have the previous version running without flaws and deployed with no issue. Can you please resolve these issues in your OVF templates.
![skadi](https://user-images.githubusercontent.com/44977426/48359962-18b26100-e66c-11e8-9fbd-86c67c4b0bae.PNG)

**Comment by orlikoski** [comment]
Thank you for bringing this up and will look into it tonight

**Comment by orlikoski** [comment]
I think I found the issue and fixed it. I was able to import both of the new OVF files (uploading now) to an ESXi 6.7 host. The upload will take another hour or so try after that to get the newest version. Once I've confirmed they are uploaded I'll push out a new release number so everyone knows it's been updated.

New Version SHA256 Hashes:
81f37559bbb68f884046e3bf9f77d34d4b7e75f24005461742ffab5b16b65793 *Skadi 2018.4.zip
b5932ef91c0014ae0e8b14fe714fedc6ea22bc457ddb97b931637eeef00264f9 *Skadi Desktop 2018.4.zip
